package.json

```diff
@@ -65,6 +66,7 @@
     "jest-cli": "^26.5.2",
     "jsdoc-to-markdown": "^7.1.0",
     "metro-react-native-babel-preset": "^0.77.0",
+    "nodemon": "^3.0.3",
```

Comment:
There are a number of file-watcher tools that could be used to do this same thing. I chose [nodemon](https://www.npmjs.com/package/nodemon) because it's tried-and-true, with almost 6MM weekly downloads, and has an easy-to-use cli.

Reply:
nodemon gets a 👍 from me